GNUmakefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: abigen
+abigen: ## Build & install abigen. Location /tools/bin/
+	./gethwrappers/generation/build_abigen
```

Comment:
can we pin the abigen version as well? Some helpful code might be https://github.com/smartcontractkit/ccip/blob/83ca5683607b9cc419891e7ede4867b467f5d4b3/tools/bin/build_abigen

Reply:
already using that build scipt from core